### ISSUE TITLE
Plate plugin

### ISSUE THREAD

**Issue by zbeyens** [tpssh/slate-paging#1]
Awesome work! I'm planning to work on a page plugin in Plate repo, would you like to collaborate?

Can I DM you on Slate slack?

**Comment by mydaoyuan** [comment]
Of course, I would love to be a collaborator, Plate is an Awesome repo🤩. 
But slate-paging is not perfect at present. 
* there are still problems in performance, 
* for some splittable block elements, such as OL and UL, it cannot be split across pages.

So this project is more of a demo, and I will continue to work to find a suitable solution.🤠

**Comment by zbeyens** [comment]
The main performance issue I have in mind is that, with 1000 pages, adding text to page 1 will move blocks from page 2 to page 1000. Do you have an idea how to tackle that?

**Comment by mydaoyuan** [comment]
> The main performance issue I have in mind is that, with 1000 pages, adding text to page 1 will move blocks from page 2 to page 1000. Do you have an idea how to tackle that?

I tested 100 pages  and used WeakMap to cache the height of the DOM, but it didn't work well.😢
